LayoutTests/imported/w3c/web-platform-tests/css/css-animations/display-none-to-display-block-dont-cancel.tentative-expected.txt

```diff
@@ -0,0 +1,3 @@
+
+PASS display:none animating to display:block should be block for the whole animation.
```

Comment:
Does this expectation file need updating? It doesn't have the output from the "Only one animation should start." assertion.

Reply:
That's how the subtest is named.